4.4.0 AppImage and Deb package installation from source - Missing Node module

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### 📝 Description

After cloning the repository and building from source on the "main" branch, I am getting a JS runtime exception for both the AppImage and .deb package.

Error message:

```sh
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'electron-positioner'
Require stack:
- /tmp/.mount_GitifyZej8vg/resources/app.asar/node_modules/menubar/lib/Menubar.js
- /tmp/.mount_GitifyZej8vg/resources/app.asar/node_modules/menubar/lib/index.js
- /tmp/.mount_GitifyZej8vg/resources/app.asar/main.js
-
    at Module._resolveFilename (internal/modules/cjs/loader.js:887:15)
    at Function.n._resolveFilename (electron/js2c/browser_init.js:257:1128)
    at Module._load (internal/modules/cjs/loader.js:732:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12913)
    at Module.require (internal/modules/cjs/loader.js:959:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/tmp/.mount_GitifyZej8vg/resources/app.asar/node_modules/menubar/lib/Menubar.js:68:45)
    at Module._compile (internal/modules/cjs/loader.js:1078:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1108:10)
    at Module.load (internal/modules/cjs/loader.js:935:32)
```

### 🪜 Steps To Reproduce

1. Clone the Git repo.
2. From the main branch, run `pnpm install`.
3. Run `npm run build && npm run make:linux`.
4. Run `sudo apt install ./dist/gitify_4.4.0_amd64.deb`.
5. Reload your shell and run `gitify` (observe the error above).
6. Run `./dist/Gitify-4.4.0.AppImage` (observe the error above).

### ℹ System Info

```Markdown
❯ cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 12 (bookworm)"
NAME="Debian GNU/Linux"
VERSION_ID="12"
VERSION="12 (bookworm)"
VERSION_CODENAME=bookworm
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```


### 📸 Screenshots

_No response_

--- a/.npmrc
+++ b/.npmrc
@@ -1,0 +1,2 @@
+node-linker=hoisted
+shamefully-hoist=true

